integration-tests/kamelet/src/test/java/org/apache/camel/quarkus/component/kamelet/it/KameletTest.java

```diff
@@ -65,4 +65,23 @@ public void testKameletChain() {
                 .statusCode(200)
                 .body(is("Hello Camel Quarkus Kamelet Chained Route"));
     }
+
+    @Test
+    public void testInvoke() {
+        RestAssured.given()
+                .contentType(ContentType.TEXT)
+                .body("Kamelet")
+                .post("/kamelet/invoke/toUpperWithBean")
+                .then()
+                .statusCode(200)
+                .body(is("KAMELET"));
+
+        RestAssured.given()
+                .contentType(ContentType.TEXT)
+                .body("Kamelet")
```

Comment:
I would send another input body so the result is different from the previous test, eg such as Kamelet2

Reply:
done